doc/luigi_patterns.rst

```diff
@@ -251,17 +251,17 @@ output folder already exists. Here is a robust version of this:
     $ [[ -d /outputs/final_output-tmp-123456 ]] && rm -r /outputs/final_output-tmp-123456
 
 Indeed, the good way is not as trivial. It involves coming up with a unique
-directory name and a pretty complex ``mv`` line, the reason ``mv`` need all
-those is because we don't want ``mv`` to move a directory into a potentially
+directory name and a more complex ``mv`` line. The reason ``mv`` needs this complexity
+is because we don't want ``mv`` to move a directory into a potentially
 existing directory. A directory could already exist in exceptional cases, for
-example when central locking fails and the same task would somehow run twice at
+example, when central locking fails and the same task would somehow run twice at
 the same time. Lastly, in the exceptional case where the file was never moved,
-one might want to remove the temporary directory that never got used.
+one might want to remove a temporary directory that never got used.
```

Comment:
Isn't it `the` because it's a specific one? I mean the process that created it knows which one it is. (My grammar is poor though, I trust you on this :))